Extension/src/LanguageServer/extension.ts

```diff
@@ -959,17 +960,64 @@ function reportMacCrashes(): void {
     }
 }
 
-let previousMacCrashData: string;
-let previousMacCrashCount: number = 0;
+export function watchForCrashes(crashDirectory: string): void {
+    if (process.platform !== "win32") {
+        prevCrashFile = "";
+        fs.stat(crashDirectory, (err) => {
+            const crashObject: Record<string, string> = {};
+            if (err?.code) {
+                // If the directory isn't there, we have a problem...
+                crashObject["fs.stat: err.code"] = err.code;
+                telemetry.logLanguageServerEvent("MacCrash", crashObject, undefined);
```

Comment:
It shouldn't be necessary to pass in `undefined` for the optional 3rd parameter.

Reply:
removed